uaclient/security.py

```diff
@@ -917,29 +936,25 @@ def prompt_for_affected_packages(
                             binary_pkg
                         )
 
-    (
-        fix_status,
-        unfixed_pkgs_released,
-        all_already_installed,
-    ) = _handle_released_package_fixes(
+    released_pkgs_install_result = _handle_released_package_fixes(
         cfg=cfg,
         src_pocket_pkgs=src_pocket_pkgs,
         binary_pocket_pkgs=binary_pocket_pkgs,
         pkg_index=pkg_index,
         num_pkgs=count,
     )
 
-    unfixed_pkgs += unfixed_pkgs_released
+    unfixed_pkgs += released_pkgs_install_result.unfixed_pkgs
 
     if unfixed_pkgs:
         print(_format_unfixed_packages_msg(unfixed_pkgs))
 
-    if fix_status:
+    if released_pkgs_install_result.fix_status:
         # fix_status is True if either:
         #  (1) we successfully installed all the packages we needed to
         #  (2) we didn't need to install any packages
         # In case (2), then all_already_installed is also True
-        if all_already_installed:
+        if released_pkgs_install_result.all_already_installed:
             # we didn't install any packages, so we're good
             print(util.handle_unicode_characters(fix_message))
             return (
```

Comment:
I think we need to change the call below to `util.should_reboot(released_pkgs_install_result.installed_pkgs)`

Reply:
Yep, great catch